dxr/static/templates/folder.html

```diff
@@ -31,14 +31,18 @@
         <th scope="col">Name</th>
         <th scope="col">Modified</th>
         <th scope="col">Size</th>
+        <th scope="col">Author</th>
```

Comment:
There's no need to add these columns here. Author isn't that important, and the VCS hash will be the same for every file under most VCSs.